Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=69dfaefee4a2dfdfee3488a306403fe1e51f0be5
Commit:     69dfaefee4a2dfdfee3488a306403fe1e51f0be5
Parent:     eb63212868c348cc6d3ec6929d7d98f7d29493e9
Author:     Takashi Iwai <tiwai@suse.de>
AuthorDate: Sat Dec 20 16:57:50 2008 +0100
Committer:  Takashi Iwai <tiwai@suse.de>
CommitDate: Sat Dec 20 16:57:50 2008 +0100

    ALSA: hda - Add quirk for another HP dv7
    
    Added the model=hp-m4 quirk for another HP dv7 (103c:30fc) with IDT
    92HD71b* codec.
    
    Reference: Novell bnc#461108
    	https://bugzilla.novell.com/show_bug.cgi?id=461108
    
    Cc: stable@kernel.org
    Signed-off-by: Takashi Iwai <tiwai@suse.de>

 sound/pci/hda/patch_sigmatel.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/sound/pci/hda/patch_sigmatel.c b/sound/pci/hda/patch_sigmatel.c
index 2cadf7c..e2941cd 100644
--- a/sound/pci/hda/patch_sigmatel.c
+++ b/sound/pci/hda/patch_sigmatel.c
@@ -1731,6 +1731,8 @@ static struct snd_pci_quirk stac92hd71bxx_cfg_tbl[] = {
 		      "HP dv5", STAC_HP_M4),
 	SND_PCI_QUIRK(PCI_VENDOR_ID_HP, 0x30f4,
 		      "HP dv7", STAC_HP_M4),
+	SND_PCI_QUIRK(PCI_VENDOR_ID_HP, 0x30fc,
+		      "HP dv7", STAC_HP_M4),
 	SND_PCI_QUIRK(PCI_VENDOR_ID_HP, 0x361a,
 				"unknown HP", STAC_HP_M4),
 	SND_PCI_QUIRK(PCI_VENDOR_ID_DELL, 0x0233,
